papas/datatypes/DefinitionsCollections.h

```diff
@@ -28,21 +28,28 @@ class Track;
 class Edge;
 class Particle;
 
-typedef std::unordered_map<Identifier, Particle> Particles;  ///< collection of Particles
-typedef std::list<Particle> ListParticles;                   ///< collection of Particles
-typedef std::unordered_map<unsigned long long, Edge> Edges;  ///< collection of Edges
+typedef std::list<Particle> ListParticles;                   ///< list of Particles
+typedef std::unordered_map<unsigned long long, Edge> Edges;  ///< collection of Edge objects
 #if WITHSORT
-typedef std::set<Identifier, std::greater<Identifier>> Ids;  ///< collection of Ids
+typedef std::set<Identifier, std::greater<Identifier>> Ids;  ///< collection containing Id objects
```

Comment:
Also I think we should really stick to the name collection for unordered maps of objects.
This is a set.